airflow/www/templates/appbuilder/baselayout.html

```diff
@@ -27,6 +27,7 @@
   {% if not appbuilder.app_theme %}
     {# airflowDefaultTheme.css file contains the styles from local bootstrap-theme.css #}
     <link href="{{ url_for_asset('airflowDefaultTheme.css') }}" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css">
```

Comment:
We need to bundle the CSS rather than rely on a external URL to support installing Airflow in environments without direct internet access.

Reply:
Agree